package.json

```diff
@@ -32,7 +32,7 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "heroku-postbuild": "echo Skip build on Heroku"
+    "postinstall": "npm run build"
   },
```

Comment:
runs the build automatically after `npm install`. Heroku will benefit from this

Reply:
Nice, thanks.  I need to learn more about package.json scripts and best practices.